Content.Server/Body/Components/SkeletonBodyManagerComponent.cs

```diff
@@ -0,0 +1,28 @@
+using Content.Server.Cloning;
+using Content.Shared.Body.Components;
+
+namespace Content.Server.Body.Components
+{
+    [RegisterComponent]
+    public sealed class SkeletonBodyManagerComponent : Component
+    {
+        /// <summary>
+        /// The dna entry used for reassembling the skeleton
+        /// updated before the entity is gibbed.
+        /// </summary>
+        [ViewVariables]
+        public ClonerDNAEntry? DNA = null;
+
+        /// <summary>
+        /// The default time it takes to reassemble itself
+        /// </summary>
+        [ViewVariables]
+        public float DoAfterTime = 5f;
+
+        /// <summary>
+        /// The list of body parts that are needed for reassembly
+        /// </summary>
+        [ViewVariables]
+        public List<SharedBodyPartComponent>? BodyParts = null;
```

Comment:
same here, store a list (or better yet hashset) of entities instead of body parts